PopupKit/Sources/PopupKit/Implementation/Views/NotificationView.swift

```diff
@@ -26,13 +26,12 @@ struct NotificationView: View {
         }
         .padding([.leading, .trailing], 15)
         .padding([.top, .bottom], 10)
-        .background(GeometryReader(content: processGeometry))
```

Comment:
Are you sure it has to be removed?

Reply:
For that, there's `gesture(dragGesture)` 
I tested it, manual removal works